cmd/capabilities.go

```diff
@@ -60,9 +60,14 @@ func recommendedCapabilitiesToBeDropped() (dropCapSet CapSet, err error) {
 func checkCapabilities(container Container, result *Result) {
 	added := CapSet{}
 	dropped := CapSet{}
+	allCapsDrop := false
 	if container.SecurityContext != nil && container.SecurityContext.Capabilities != nil {
 		added = NewCapSetFromArray(container.SecurityContext.Capabilities.Add)
-		dropped = NewCapSetFromArray(container.SecurityContext.Capabilities.Drop)
+		if len(container.SecurityContext.Capabilities.Drop) != 0 && string(container.SecurityContext.Capabilities.Drop[0]) == "all" {
```

Comment:
Thanks for the quick fix!

It will be better to `strings.ToLower(string(container.SecurityContext.Capabilities.Drop[0]))`.

Unfortunately I cannot find proper refs anywhere in k8s docs or code but my tests show that this `ALL` keyword is case insensitive...